### ISSUE TITLE
Support lz4_flex for IPC

### ISSUE THREAD

**Issue by ghuls** [jorgecarleitao/arrow2#986]
Support lz4_flex for IPC, similarly as was done for parquet, so IPC with at least lz4 compression can be used when compiling for WASM:
https://github.com/jorgecarleitao/parquet2/pull/124/commits/e01c78f85434177b78cf1cf876eb9a2fa41bc9c3

**Comment by jorgecarleitao** [comment]
agreed. Marking as a good first issue / help wanted. I think it is a cool exercise. I can mentor if help is required.
